source/_integrations/google_drive.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Google Drive
+description: Instructions on how to use Google Drive in Home Assistant.
+ha_category:
+  - Backup
+ha_iot_class: Cloud Polling
+ha_release: '2025.2'
+ha_config_flow: true
+ha_domain: google_drive
+ha_codeowners:
+  - '@tronikos'
+ha_integration_type: service
+google_dev_console_link: https://console.developers.google.com/start/api?id=drive
+api: Google Drive API
+api_link: https://console.developers.google.com/start/api?id=drive
+---
+
+This integration allows you to connect your [Google Drive](https://drive.google.com) with Home Assistant Backups. When you set up this integration, your Google Drive will have a new folder called Home Assistant where all the backups will be stored. You can rename this folder to whatever you like in Google Drive at any point of time. If you delete the folder, it will automatically be re-created as long as you have the integration enabled.

```

Comment:
```suggestion
This {% term integration %} allows you to connect your [Google Drive](https://drive.google.com) with Home Assistant Backups. When you set up this integration, your Google Drive will have a new folder called Home Assistant where all the backups will be stored. You can rename this folder to whatever you like in Google Drive at any point in time. If you delete the folder, it will automatically be re-created as long as you have the {% term integration %} enabled.

```